subsys/bluetooth/controller/Kconfig.df

```diff
@@ -0,0 +1,119 @@
+# Zephyr Bluetooth Controller configuration options
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTLR_DF_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_TX_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_RX_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_RX_1US_SUPPORT
+	bool
+
+config BT_CTLR_DF_ANT_SWITCHING_2US_SUPPORT
+	bool
+
+config BT_CTLR_DF_ANT_SWITCHING_1US_SUPPORT
+	bool
+
+menuconfig BT_CTLR_DF
```

Comment:
Now there is a menuconfig and a config named `BT_CTLR_DF`. That is confusing and probably not intentional.

Reply:
Duplicated config removed. Left menuconfig.